api/base/serializers.py

```diff
@@ -82,8 +82,9 @@ class ConditionalField(ser.Field):
 
     def __init__(self, field, **kwargs):
         super(ConditionalField, self).__init__(**kwargs)
-        self.field = field
-        self.source = self.field.source
+        self.field = getattr(field, 'child_relation', field)
+        # source is intentionally field.source and not self.field.source
+        self.source = field.source
```

Comment:
For binding purposes, otherwise you get a `ValueError: too many values to unpack`.